buildflow/api/options.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class StreamingOptions:
```

Comment:
nit: Can we just make this a general RuntimeOptions / AutoscalerConfig object? I could see needing this for batch when running on k8